### PR TITLE
Fixed the navbar 

### PR DESCRIPTION
**Pull request by sandrineay**
With two different views based on whether the user is signed in or not plus a log out link (YEAY)

### CODE REVIEW

File: app/views/shared/_navbar.html.erb [hunk 1]
```diff
@@ -1,59 +1,63 @@
...
-        <div class="badge badge-blue">3</div>
+      <!-- Profile picture and dropdown -->
+      <div class="navbar-wagon-item">
+        <div class="dropdown">
+          <img src="http://placehold.it/50x50" class="avatar dropdown-toggle" id="navbar-wagon-menu" data-toggle="dropdown">
+          <ul class="dropdown-menu dropdown-menu-right navbar-wagon-dropdown-menu">
+            <li><a href="#">Profile</a></li>
+            <li><a href="#">Dashboard</a></li>
+            <li>
+              <%= link_to destroy_user_session_path, method: :delete do %>
```
Comment (tunihoop) [inline app/views/shared/_navbar.html.erb]: Suggestion to simplify the 3 lines 40-42:
<%= link_to "Log out", destroy_user_session_path, method: :delete %>

File: app/views/shared/_navbar.html.erb [hunk 2]
```diff
@@ -1,59 +1,63 @@
...
+        </button>
+      </form>
+    <% end %>
+  </div>
 
   <!-- Right Navigation -->
   <div class="navbar-wagon-right hidden-xs hidden-sm">
+    <% if user_signed_in? %>
+      <!-- Link to Articles owned by the user -->
+      <a href="" class="navbar-wagon-item navbar-wagon-link">Articles</a>
```
Comment (tunihoop) [inline app/views/shared/_navbar.html.erb]: Here, we can add the link to the user bookings:
<%= link_to "Articles", user_bookings_path(current_user), class: "navbar-wagon-item navbar-wagon-link" %>
We also need to change the comment to "Link to Articles booked by the user"

File: app/views/shared/_navbar.html.erb (hunk 1 above)
Comment (sandrineay) [inline app/views/shared/_navbar.html.erb]: Thanks! Done :)

File: app/views/shared/_navbar.html.erb [hunk 3]
```diff
@@ -1,59 +1,63 @@
...
-      </button>
-    </form> -->
-
-    <!-- Text link -->
-    <a href="" class="navbar-wagon-item navbar-wagon-link">Share your Articles</a>
-
-    <!-- Text link -->
-    <a href="" class="navbar-wagon-item navbar-wagon-link">Help</a>
+      <!-- Link to Articles the user has booked previously -->
+      <a href="" class="navbar-wagon-item navbar-wagon-link">Bookings</a>
```
Comment (tunihoop) [inline app/views/shared/_navbar.html.erb]: Here we can direct the user to the path:
<%= link_to "Bookings", user_bookings_path(@user) %>

File: app/views/shared/_navbar.html.erb (hunk 2 above)
Comment (tunihoop) [inline app/views/shared/_navbar.html.erb]: Here we can direct the user to the path:
<%= link_to "Articles", admin_articles_path(@user) %>

**Pull request merged**